ibm-messaging-mq-cloud-showcase-app/backend/AWS-showcase-backend/models/Responder.js

```diff
@@ -34,12 +34,27 @@ class Responder {
             this.mqclient.get(queueName,1,null, 'DYNAMIC', this.appId)
             .then((messages) => {
                 debug_warn(`The producer retrieved this message ${JSON.stringify(messages)}`);
-                if(!messages[0].replyToMsg) {
+                if (messages.length === 0) {
+                    debug_info("Poison Message");
+                    resolve();
```

Comment:
line 39 is resolving empty, 42 is resolving null. Do they have a different meaning?

Shouldn't 42 be resolving so the message is removed from the queue.